adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -335,8 +334,11 @@ class Transform(protected val args: TransformArgs) extends BDGSparkCommand[Trans
       (rdd ++ t.rdd, sd ++ t.sequences, rgd ++ t.recordGroups)
     })
 
+    // make a new alignment record rdd
+    val newRdd = AlignedReadRDD(mergedRdd, mergedSd, mergedRgd)
```

Comment:
alignment record vs. aligned read?  I need to go see what the difference is.  The comment and classname should match.